Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/viewholder/InputBindingSettingViewHolder.java

```diff
@@ -37,13 +32,12 @@ protected void findViews(View root)
   @Override
   public void bind(SettingsItem item)
   {
-    SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences(mContext);
-
     mItem = (InputBindingSetting) item;
 
     mTextSettingName.setText(mItem.getNameId());
-    mTextSettingDescription
-            .setText(sharedPreferences.getString(mItem.getKey() + mItem.getGameId(), ""));
+    mTextSettingDescription.setText(AdHocStringSetting
+            .getStringGlobal(Settings.FILE_DOLPHIN, Settings.SECTION_BINDINGS,
+                    mItem.getKey() + "Description", ""));
```

Comment:
I think it would make sense to create a method in `InputBindingSetting` that does the call to `getString`, so that the code here doesn't need to hardcode any section names or such.